src/lib/y2storage/bcache_cset.rb

```diff
@@ -67,6 +67,20 @@ def display_name
       format(_("Cache set (%s)"), devices)
     end
 
+    # Returns list of devices that can be used for caching. Like existing caching sets or
+    # suitable block devics.
+    def self.possible_devices(device_graph)
```

Comment:
Same than above, I would expect this in the action. After all, it's about what devices to offer in the form.

Reply:
well, one view is that it is devices for form, another view can be that it is devices where bcache can be created without destruction of data. But I can move it.

Reply:
Not really. For example, here we include devices that contain a filesystem as long as that filesystem has no mount point assigned currently. That can imply a very severe destruction of data, but it's ok to offer such devices from the Partitioner point of view.